addon/templates/components/frost-sidebar.hbs

```diff
@@ -1,2 +1,17 @@
-ember-frost-sidebar component
-{{yield}}
\ No newline at end of file
+{{#liquid-if isOpen class="container" }}
+    <div class="open-container">
+        <div class="pod">
+			{{yield}}
+        </div>
+
+        <div class="tab tab-open" {{action (action 'toggleSidebar')}}>
+			{{frost-svg class="icon" path="frost/sidebar-layer-open"}}
```

Comment:
The component is different.  You are using frost-svg here and it should be frost-icon.  Take a look at ember-frost-core and how it is being used in the many demo apps in there.